packages/@o3r/testing/src/localization/localization-mock.ts

```diff
@@ -17,10 +17,7 @@ export class TranslatePipeMock implements PipeTransform {
   }
 }
 
-@NgModule({
-  declarations: [TranslatePipeMock],
-  exports: [TranslatePipeMock]
-})
+@NgModule()
```

Comment:
On this version of Angular (pre-module compilation) was it possible to declare a pipe coming from a built library?
If not you may need to provide an additional module declaring these removed pipes

Reply:
In fact outside the module you will have the pipe from the common module and not the overrided one without the export